opl/cluster_read.py

```diff
@@ -373,6 +373,42 @@ def measure(self, ri, name, command, output="text"):
         return name, result
 
 
+class CountLinePlugin(BasePlugin):
+    def measure(self, ri, name, command, output="text"):
+        """
+        Execute command "command" and return result as per its "output" configuration
+        """
+        error_count = 0
+        warning_count = 0
+        line_count = 0
+        
+        # Execute the command
+        result = execute(command)
+        result_lines = result.splitlines()
+
+        for line in result_lines:
+            line_count+=1
+            if line["level"] == "error":
+                error_count+=1
+            elif line["level"] == "warning":
+                warning_count+=1
+        
+        final_result = {
+        "measurements": 
+            { "logs": 
+                {"openshift-pipelines": 
+                    {"pipelines-as-code-controller": 
+                        {"all": line_count,
+                        "error": error_count,
+                        "warning": warning_count,
+                    }
+                }
+            }
+        }
+        }
```

Comment:
Plugin should return just a `name, value` tuple which would be something like `name, {"all": line_count, ...}` AFAICT